includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1390,6 +1391,15 @@ private function get_payment_methods_from_request() {
 	public function set_payment_method_title_for_order( $order, $payment_method_type, $payment_method_details ) {
 	}
```

Comment:
Do we still need this base definition of the function that is never overriden with this signature?

Reply:
Removed in https://github.com/Automattic/woocommerce-payments/pull/6826/commits/11ac6ed5014ee3bf6efffa8d5b20223df7d2a96a